server/routes/index.js

```diff
@@ -0,0 +1,34 @@
+var controller = require('../controllers');
+var helpers = require('../helpers')
+
+module.exports = function(app, express) {
+	app.get('/',function(req,res){
```

Comment:
Is this route being used?  Is this conflicting with the `app.use(express.static...)`